File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float in;
    do
    {
        in = get_float("Change: ");
    }
    while (in < 0);

    // Convert to cents
    int coins = 0, change = round(in*100);

    coins += (change / 25);
    change %= 25;

    coins += (change / 10);
    change %= 10;

    coins += (change / 5);
    change %= 5;

    coins += change;

    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

n = get_float("Change owed: ")
while (n < 0):
    n = get_float("Change owed: ")

coins, cents = 0, int(n * 100)

coins = coins + (cents // 25)
cents = cents % 25

coins = coins + (cents // 10)
cents = cents % 10

coins = coins + (cents // 5)
cents = cents % 5

coins = coins + cents

print(coins)